src/backend/main.ts

```diff
@@ -980,17 +991,19 @@ ipcMain.handle(
     )
     return game
       .launch(launchArguments)
-      .catch((error) => {
-        logError(error.stack ?? `${error}`, LogPrefix.Backend)
+      .catch((exception: Error) => {
```

Comment:
You can't know for sure that what you got here is an `Error`, throwing *anything* is fine
```
$ node
Welcome to Node.js v16.16.0.
Type ".help" for more information.
> throw 'Hey'
Uncaught 'Hey'
```